tensorflow-framework/tensorflow-data.md

```diff
@@ -0,0 +1,209 @@
+
+NOTE: This readme follows the discussion of this [RFC]()
+
+
+Tensorflow-Data (Java)
+==
+
+TensorFlow Data provides utilities and APIs for loading data of various formats, and preparing datasets for use in training and using deep learning models
+. This package
+ provides a
+ simple API for configuring and iterating over
+ datasets in both "graph" and "eager" mode.
+
+Usage
+--
+
+The `Dataset` class represents a sequence of elements which can be iterated over and
+transformed. Each element is a list of "output" operands, represented by the type `List<Output<?>>`. 
```

Comment:
again, is it `List<Output<?>>` or `List<Operand<?>>`?

Reply:
It's `List<Output<?>>`

Reply:
Now it's `List<Operand<?>>`

Reply:
Now it's `List<Operand<?>>`